config/locales/zh-CN.yml

```
@@ -442,14 +448,14 @@ zh-CN:
     adoptions:
       no_ownership_calls:
       no_ownership_requests:
-      title:
+      title: 领养
       subtitle_html:
     edit:
-      change_avatar:
+      change_avatar: 修改头像
       email_awaiting_confirmation: 请验证你的新邮箱地址 %{unconfirmed_email}
       enter_password: 输入密码
-      hide_email: 在公开的个人资料里面隐藏我的 Email
-      optional_full_name:
+      hide_email: 在公开的个人资料里面隐藏我的邮件
```

No translation is necessary as ‘Email’ is acceptable untranslated. This suggestion is for consistency.
```suggestion
      hide_email: 在公开的个人资料里面隐藏我的邮箱
```